### DISCUSSION TITLE
Any contributors focused on new user experience?

### DISCUSSION THREAD

**Discussion by samohd** [FreeCAD/FreeCAD#16315] [Code contribution discussions]
Are there any contributors with a focus on new user experience? Or is everyone contributing a long time user with tons of experience that has lost sight of how confusing the program is for someone new, Such as how common sense things that don't happen in a common sense way? Assembly is a good example of this. I feel like the dev team is missing someone looking out for newbies. And from my perspective the potential new users count is much higher than the already using count. How many gazillions of users of Tinker CAD... seek out more capability and bypass FreeCAD, because of confusion?

**Comment by luzpaz** [comment]
There is a FreeCAD Design Workgroup. You can find them on the discord server.
